eng/scripts/Update-Snippets.ps1

```diff
@@ -11,13 +11,7 @@ param (
     [string] $SnippetToolPath=''
 )
 
-$root = "$PSScriptRoot/../../sdk"
-
-# special casing * here because single invocation of SnippetGenerator is much faster than
-# running it per service directory
-if ($ServiceDirectory -and ($ServiceDirectory -ne "*")) {
-    $root += '/' + $ServiceDirectory
-}
+$root = "$PSScriptRoot/../../sdk/$ServiceDirectory"
```

Comment:
Update snippets script doesn't run on everything anymore, so simplify the logic here.